misc/ls-extensions/modules/bal-shell-service/src/main/java/io/ballerina/shell/service/util/TypeUtils.java

```diff
@@ -51,4 +53,33 @@ public static String getMimeTypeFromName(int typeTag) {
                 return Constants.MIME_TYPE_PLAIN_TEXT;
         }
     }
+
+    /**
+     * Returns json string for a given object if that object is accepted as json convertible type
+     * otherwise returns the string representation for the object.
+     *
+     * @param value object needs to be converted
+     * @return converted string
+     */
+    public static String convertToJsonIfAcceptable(Object value) {
+        Type type = io.ballerina.runtime.api.utils.TypeUtils.getType(value);
```

Comment:
Another place where we can replace the fully qualified name if required.

Reply:
This is because the class name `TypeUtils` is common here. So, on the other way we can import the function `io.ballerina.runtime.api.utils.TypeUtils.getType` and use. 
The same happens for the `JsonUtils` as well. 
I'll change the other places.